index.html

```diff
@@ -59,7 +59,7 @@ <h1>
         </div>
     </header>
 
-    <div class="container">
+    <div class="container" style="background-size: 100%; max-width: 100%; background-image:url('images/bag.jpg');">
```

Comment:
try to not use inline styling. You can create a `.container` css class and put all of those there.